docs/index.html

```diff
@@ -70,16 +64,81 @@ <h3>Example 3</h3>
   &lt;body&gt;
     &lt;script&gt;
       bootstrapGrist({
-        initialFile: 'investment-research.grist',   // path to .grist
-        name: 'Investment Research',                // optional name
+        initialFile: 'weresheet.grist',   // optional path to .grist
+        name: 'Werewolf Fact Sheet',      // optional name
+        elementId: 'were-div',            // optional element name
+        light: true                       // optional minimalist style
```

Comment:
I'm noticing now that `light: true` doesn't feel very readable. It sounds like it's turning off dark mode. `embed: true` would match what actually happens in the code and seems slightly more understandable but not great. Maybe `minimalUI: true`?

Reply:
I agree the name isn't great. It is usable without embedding (i.e. when Grist takes over whole page) which is why I avoided `embed`, which I would have preferred otherwise since it is a query parameter we have documented https://support.getgrist.com/FAQ/#can-i-embed-grist-into-my-website and it would have been nice to "rhyme" with it.

`minimalUI` would be ok. How would you feel about `thin` or `lightweight` or (consults chatbot) `svelte`, `ethereal`, `dainty`  ... ? Or we could focus on the "single page" aspect of the bundle of changes with `singlePage`?

Reply:
I think I like `singlePage` best.

I'm actually confused. I thought that with this code:

```
  if (gristOverrides.light) {
    location.searchParams.set('embed', 'true');
    location.searchParams.set('style', 'light');
  }
```

Setting `embed` would make the doc read only, but it's not.

Reply:
Renamed to `singlePage` (though PR isn't updating right now)
![Screenshot from 2023-07-28 11-20-48](https://github.com/gristlabs/grist-static/assets/118367/8454fa52-d720-47d0-9ef8-bca88d4a6338)

Yes the `embed` thing is a bit funky.

Reply:
Do you know why it doesn't make things read-only?

Reply:
I was afraid you were going to ask that :). I'll investigate.

Reply:
Turns out it is because the doc technically qualifies as a fork currently, and the read-only logic has an exception for forks: https://github.com/gristlabs/grist-core/blob/3d84014116a3f07010131ef9989073e763e3031d/app/common/gristUrls.ts#L399-L400

If we change the doc setup later (I expect we will, the relevant code is very hacky), then the read-only mode could kick in - but it'd be easy to fix by specifying an explicit mode.